src/api/controllers/articleTagController.js

```diff
@@ -0,0 +1,99 @@
+import _ from 'lodash';
+import Sequelize from 'sequelize';
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+import errorMessages from '../../helpers/constants/error.messages';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Tag, ArticleTag, Article } = models;
+
+/**
+ *
+ * @export
+ * @class articleTags
+ */
+export default class articleTags {
+  /**
+   * A method to all tags of an article
```

Comment:
@nkpremices This comment needs some editing.

Reply:
Noted @MemunaHaruna

Reply:
It's done @MemunaHaruna